.gitignore

```diff
@@ -27,6 +27,8 @@ broadcast/*/31337/
 target
 elf
 artifacts
+bacon.toml
```

Comment:
A bit unrelated, but added these since I am experimenting with running bacon-ls as a wrapper for rust-analyzer to improve incremental compilation.